ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/PDEPlugin.java

```diff
@@ -42,6 +43,7 @@
 import org.eclipse.ui.internal.views.log.ILogFileProvider;
 import org.eclipse.ui.internal.views.log.LogFilesManager;
 import org.eclipse.ui.plugin.AbstractUIPlugin;
+import org.eclipse.ui.progress.IProgressService;
```

Comment:
I wonder why there is `org.eclipse.ui.progress.IProgressService` and `org.eclipse.e4.ui.progress.IProgressService` which looks very similar.

Reply:
One is the "old" eclipse.ui and the other the "new" rcp E4 ... but I think the E4 one is not really completed and would need a declarative way (e4xmi) to add icons for job groups.